include/es/pooled_resource.h

```diff
@@ -20,10 +20,6 @@
 template <class T>
 class pooled_resource {
  public:
-    ~pooled_resource()
-        {
```

Comment:
I'm pretty sure that I added that destructor yesterday when I was trying to kill a segfault.... I don't think it was doing anything useful, so it seemed better to put it back the way it was.